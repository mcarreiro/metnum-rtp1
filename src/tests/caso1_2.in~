500 500 2 5 400 4 
400 250
250 400
100 250
250 100
