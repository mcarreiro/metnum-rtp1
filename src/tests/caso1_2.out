0	0	-100.00000
0	1	-100.00000
0	2	-100.00000
0	3	-100.00000
0	4	-100.00000
1	0	-100.00000
1	1	400.00000
1	2	233.33333
1	3	400.00000
1	4	-100.00000
2	0	-100.00000
2	1	233.33333
2	2	233.33333
2	3	233.33333
2	4	-100.00000
3	0	-100.00000
3	1	400.00000
3	2	233.33333
3	3	400.00000
3	4	-100.00000
4	0	-100.00000
4	1	-100.00000
4	2	-100.00000
4	3	-100.00000
4	4	-100.00000
